opal/util/info_subscriber.c

```diff
@@ -348,6 +348,12 @@ int opal_infosubscribe_subscribe(opal_infosubscriber_t *object, char *key, char
     opal_hash_table_t *table = &object->s_subscriber_table;
     opal_callback_list_item_t *callback_list_item;
 
+    if (strlen(key) > OPAL_MAX_INFO_KEY-strlen("__IN_")) {
```

Comment:
Is this a `opal_show_help()`-worthy error, or should it be an abort/die-worthy error (because it's an OMPI developer programmer error)?

Reply:
Not sure, all I did so far for this one was make it say "DEVELOPER WARNING"